projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.ts

```diff
@@ -77,9 +77,98 @@ export class IgxSplitBarComponent {
         return this.siblings[0].hidden && !this.siblings[1].hidden;
     }
 
+    get collapseNextIcon() {
+        return this.type === SplitterType.Horizontal ? 'arrow_right' : 'arrow_drop_down';
+    }
+
+    get collapsePrevIcon() {
+        return this.type === SplitterType.Horizontal ? 'arrow_left' : 'arrow_drop_up';
+    }
+
     /**
-     * @hidden @internal
+     * A field that holds the initial size of the main `IgxSplitterPaneComponent` in each couple of panes devided by a gripper.
+     * @private
+     * @memberof SplitterComponent
+     */
+    private initialPaneSize!: number;
+
+    /**
+     * A field that holds the initial size of the sibling `IgxSplitterPaneComponent` in each couple of panes devided by a gripper.
+     * @private
+     * @memberof SplitterComponent
      */
+    private initialSiblingSize!: number;
+
+    /**
+     * The sibling `IgxSplitterPaneComponent` in each couple of panes devided by a gripper.
+     * @private
+     * @memberof SplitterComponent
+     */
+    private sibling!: IgxSplitterPaneComponent;
+
+    /**
+     * @hidden
+     * @internal
+     */
+    @HostListener('keydown', ['$event'])
+    keyEvent(event: KeyboardEvent) {
+        const key = event.key.toLowerCase();
+        const ctrl = event.ctrlKey;
+        event.stopPropagation();
+            switch (key) {
+                case 'arrowup':
+                    if (this.type === 1) {
+                        if (ctrl) {
+                            this.pane.hidden ? this.onCollapsing(true) : this.onCollapsing(false);
```

Comment:
Isn't Ctrl+ArrowUp/Ctrl+ArrowLeft  equivalent to clicking the previous button? So shouldn't it call onCollapsing(false) in both cases?